### ISSUE TITLE
Run tests based on type of change

### ISSUE THREAD

**Issue by vinayak-mehta** [camelot-dev/camelot#521]
We shouldn't run code tests for doc changes.

**Comment by bosd** [comment]
True, those can be skipped on GH actions. But the docs build / check and linting still could be run. 


The cookiecutter used here. Just didn't make a distinction between code changes or doc changes.
